t/README.md

```diff
@@ -0,0 +1,22 @@
+# UNIT TESTS
+
+## Introduction
```

Comment:
I think this fits better in `CONTRIBUTING.md`.

Reply:
Done